qa/tasks/vstart_runner.py

```diff
@@ -160,8 +160,8 @@ class LocalRemoteProcess(object):
     def __init__(self, args, subproc, check_status, stdout, stderr):
         self.args = args
         self.subproc = subproc
-        self.stdout = stdout or BytesIO()
-        self.stderr = stderr or BytesIO()
+        self.stdout = stdout or StringIO()
```

Comment:
the title of the commit message reads

> qa/cephfs: to be python3 compatible

but what this commit changes is `qa/tasks/vstart_runner.py`.

and i cannot find the analysis or the explanation in the commit message. please expand the commit message with some rationales of this change.

Reply:
Will add more detail info in the commit comment later. Thanks.